### ISSUE TITLE
Adb.authenticate fails

### ISSUE THREAD

**Issue by stylesuxx** [yume-chan/ya-webadb#404]
### Issue Checklist

- [X] I'm using the library programmatically

### Library version

0.0.12

### Runtime

Linux, Chrome Version 100.0.4896.127 (Official Build) (64-bit)

### Device

DJI FPV goggles V2

### Describe the bug

I am having troubles with authentification - interestingly enough with the Demo application everything works fine and I simply don't see what I could be doing wrong, it boils down to this:

```javascript
const credentialStore = new AdbWebCredentialStore();
const device = await AdbWebUsbBackend.requestDevice();
const streams = await device.connect();
const adb = await Adb.authenticate(streams, credentialStore, undefined);
```

The last line throws an error:
```
RangeError: Offset is outside the bounds of the DataView
    at DataView.getUint32 (<anonymous>)
    at number.ts:92:1
    at tryCatcher (bluebird.js:5370:1)
    at Promise._settlePromiseFromHandler (bluebird.js:3366:1)
    at Promise._settlePromise (bluebird.js:3423:1)
    at Promise._settlePromise0 (bluebird.js:3468:1)
    at Promise._settlePromises (bluebird.js:3548:1)
    at _drainQueueStep (bluebird.js:145:1)
    at _drainQueue (bluebird.js:138:1)
    at Async._drainQueues (bluebird.js:154:1)
    at Async.drainQueues (bluebird.js:67:1)
```

Stack trace was a bit useless so I stepped through it and this is the line where it throws => https://github.com/yume-chan/ya-webadb/blob/e946b455b143464b40fabd2c24feaf59528b404d/libraries/adb/src/adb.ts#L100

Those are my library dependencies:
```
"@yume-chan/adb": "^0.0.12",
"@yume-chan/adb-backend-webusb": "^0.0.12",
"@yume-chan/adb-credential-web": "^0.0.12",
```

### Steps to reproduce

```javascript
const credentialStore = new AdbWebCredentialStore();
const device = await AdbWebUsbBackend.requestDevice();
const streams = await device.connect();
const adb = await Adb.authenticate(streams, credentialStore, undefined);
```

**Comment by yume-chan** [comment]
It was reported and fixed in #395. I have released version 0.0.13 including the fix, please try.

**Comment by stylesuxx** [comment]
Can confirm with 0.0.13 everything is working - thank you for the quick new release.
